### PR TITLE
create depfinder based audit system

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@regro/auto-tick this is mostly ready for review, I'll add a test but the functionality is there.

**Comment by CJ-Wright** [comment]
@beckermr @scopatz

File: conda_forge_tick/audit.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import json
```
Comment (scopatz) [inline conda_forge_tick/audit.py:1]: Please add a module-level docstring

**Comment by scopatz** [comment]
LGTM. Thanks @CJ-Wright!

File: conda_forge_tick/audit.py (hunk 1 above)
Comment (CJ-Wright) [inline conda_forge_tick/audit.py:1]:
```suggestion
"""Audit the dependencies of the conda-forge ecosystem"""
import json
```

**Comment by beckermr** [comment]
LGTM!

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**